packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -10,7 +10,7 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { ServerContext, ServerContextValue } from '@eclipse-sirius/sirius-components-core';
+import { ServerContext, ServerContextValue, theme } from '@eclipse-sirius/sirius-components-core';
```

Comment:
The `theme` object imported here is just an helper provided to applications which should create their own theme. This is not the current theme of the application and thus should not be used internally.

You are not using here, the theme of the application.